keps/sig-scheduling/902-non-preempting-priorityclass/README.md

```diff
@@ -146,13 +146,110 @@ and are not requesting additional changes.
 Add basic e2e tests, to ensure all components are working together.
 
 Ensure existing tests (for preempting PriorityClasses) do not break.
 
+### Graduation Criteria
+#### Alpha (v1.15):
+
+- [x] Support NonPreemptingPriority in PriorityClasses
+
+#### Beta (v1.19):
+
+- [ ] Add integration test for NonPreemptingPriority.
+- [ ] Graduate NonPreemptingPriority to Beta.
+- [ ] Update documents to reflect the changes.
+
+
+
+## Production Readiness Review Questionnaire
+
+### Feature enablement and rollback
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [x] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: NonPreemptingPriority
+    - Components depending on the feature gate: 
+      - kube-apiserver 
+      - kube-scheduler
+  - [ ] Other
+    - Describe the mechanism:
+    - Will enabling / disabling the feature require downtime of the control
+      plane?
+    - Will enabling / disabling the feature require downtime or reprovisioning
+      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
+
+* **Does enabling the feature change any default behavior?**
+  Yes
+
+* **Can the feature be disabled once it has been enabled (i.e. can we rollback
+  the enablement)?**
+  Yes
+
+* **What happens if we reenable the feature if it was previously rolled back?**
+  If the Pod with NonPreempt is still pending, it will preempt the other pods with lower priority in the new scheduling cycle.
+
+* **Are there any tests for feature enablement/disablement?**
+  Yes
+
+### Rollout, Upgrade and Rollback Planning
+* **How can a rollout fail? Can it impact already running workloads?**
+  The scheduler errors and exits during start up. Existing workloads are not
+  affected.
+
+* **What specific metrics should inform a rollback?**
+  N/A.
+
+* **Were upgrade and rollback tested? Was upgrade->downgrade->upgrade path tested?**
+  N/A.
+
+* **Is the rollout accompanied by any deprecations and/or removals of features?**
+  N/A.
+
+### Scalability
+* **Will enabling / using this feature result in any new API calls?**
+  No
+
+* **Will enabling / using this feature result in introducing new API types?**
+  No
+
+* **Will enabling / using this feature result in any new calls to cloud
+  provider?**
+  No
+
+* **Will enabling / using this feature result in increasing size or count
+  of the existing API objects?**
+  No
+
+* **Will enabling / using this feature result in increasing time taken by any
+  operations covered by [existing SLIs/SLOs][]?**
+  No
+
+* **Will enabling / using this feature result in non-negligible increase of
+  resource usage (CPU, RAM, disk, IO, ...) in any components?**
+  No
+
+### Troubleshooting
+* **How does this feature react if the API server and/or etcd is unavailable?**
+  N/A.
+
+* **What are other known failure modes?**
+  N/A.
+
+* **What steps should be taken if SLOs are not being met to determine the problem?**
+Errors for the preempt process are visible in logs.
```

Comment:
We can also mention that some metrics (like total_preemption_attempts) can provide clues.